src/Microsoft.Spatial/Gml/GmlReader.cs

```diff
@@ -708,6 +776,17 @@ private bool ReadStartOrEmptyElement(string element)
                 return !isEmptyElement;
             }
 
+            private int? ReadAttribute()

```

Comment:
Curious where or when this is applied...

Reply:
Rollbacked

---
In reply to: [516195715](https://github.com/OData/odata.net/pull/1915#discussion_r516195715) [](ancestors = 516195715)